.gitlab-ci.yml

```diff
@@ -315,10 +315,15 @@ test:acceptance:script:
     - apt update && apt install -yyq awscli
     # Lock the bucket to block concurrent jobs
     - while aws s3 ls s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/lock; do
-    -   echo "$S3_BUCKET_REPO_PATH locked, waiting..."
+    -   rm -f /tmp/lock
```

Comment:
This line is unnecessary

Reply:
woops. nice catch! it was supposed to be `/tmp/lock.in` . fixed.

Reply:
I added one more removal and an echo with waiting, which was there, so I did not try to smuggle too much 😘